tools/test-proxy/Azure.Sdk.Tools.TestProxy/Properties/launchSettings.json

```diff
@@ -2,6 +2,7 @@
   "profiles": {
     "Azure.Sdk.Tools.TestProxy": {
       "commandName": "Project",
+      "commandLineArgs": "start --storage-location=C:/repo/azure-sdk-for-net",
```

Comment:
Did you intend to have your local paths here?

Reply:
Frustration. Def shouldn't be present. Thanks for the catch Wes.